shop-service/src/functions/getProductsById/mock.json

```diff
@@ -0,0 +1,3 @@
+{ 
+  "pathParameters": { "productId" : "1b9d6bcd-bbfd-4b2d-9b5d-ab8dfbbd4bed" }
+}
```

Comment:
This mock should be removed in this task

Reply:
I think it is used is for autoswagger